controllers/image_generator.go

```diff
@@ -308,6 +308,15 @@ func (r *ImageGenerator) createJobFromFile(jobFileName string) (*batchv1.Job, er
 		job.Spec.Template.Spec.Containers[0].Image = podvmBuilderImage
 	}
 
+	// If RELATED_PODVM_PAYLOAD_IMAGE environment variable is set, use it
+	// Otherwise, use the default podvm payload image
+	// There is only one initContainer in the job, so we don't need to check the container name
+	podvmPayloadImage := os.Getenv("RELATED_IMAGE_PODVM_PAYLOAD")
```

Comment:
Would it work if it's being set in the podvm-image-cm? IIUC it's not, and i think it may be a bit confusing that this is image related configuration only but it's set differently

Reply:
The RELATED_* is also used for disconnected installs (useful for mirroring). Hence we use the same thing.